src/Components/Tasks/Tasks.tsx

```diff
@@ -1,7 +1,70 @@
-import React from 'react'
+import React, { useContext, useEffect, useState } from "react";
+import { useNavigate } from "react-router";
+import { AuthContext } from "../../Context/AuthContext";
+// import noData from './../../assets/images/no-data.png'
+import axios from "axios";
 
 export default function Tasks() {
+  const [tasks, setTasks] = useState([]);
+
+  const { baseUrl, requestHeaders }: any = useContext(AuthContext);
+  const navigate = useNavigate();
+
+  const navigateToNew = () => {
+    navigate("/dashboard/add-task");
+  };
+
+  let getTasksList = () => {
+    axios
+      .get(`${baseUrl}/Task`, { headers: requestHeaders })
+      .then((response: any) => {
+        console.log("sucsess", response);
```

Comment:
you need to remove console logs from the branch before merging it into product branch, to avoid revealing any information in the console to application users

Reply:
done